install.rst

```diff
@@ -18,49 +18,136 @@
 Installing Zeek
 ===============
 
-Prerequisites
+To run Zeek, grab our official Docker images, download our Linux binary
+packages, install via Homebrew on your Mac, use the ports collection on FreeBSD,
+or build Zeek yourself.
+
+Docker Images
 =============
 
-Before installing Zeek, you'll need to ensure that some dependencies
-are in place.
+We provide `official Docker images <https://hub.docker.com/u/zeekurity>`_ on the
```

Comment:
```suggestion
We provide `official Docker images <https://hub.docker.com/u/zeekurity>`_ on
```

Also wonder if `zeekurity` should shine through here already than being hidden behind "official Docker images" link. While it may be obvious to the initiated, "zeekurity" is only later mentioned in the pull commands, but otherwise doesn't seem to be referenced in the in the zeek-docs tree. For someone new, the username/repo name may be surprising (much lesser if it was just `zeek`, but that seems taken).

Maybe something like, "we provide official docker images in the zeekurity/zeek repo on Docker Hub..." ?

Reply:
Done